CHANGELOG.md

```diff
@@ -39,7 +39,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ---
 
-## [v1.11.0](https://github.com/provenance-io/provenance/releases/tag/v1.11.0) - 2022-06-13
+## [v1.11.1-rc1](https://github.com/provenance-io/provenance/releases/tag/v1.11.1-rc1) - 2022-06-14
```

Comment:
We either need to completely remove the `v1.11.0` tag or create another section for `v1.11.1` with changes that occurred between `v1.11.0` and `v1.11.1`.